doc/api/assert.md

```diff
@@ -43,12 +43,11 @@ Primitive values are compared with the [Abstract Equality Comparison][]
 
 Only [enumerable "own" properties][] are considered. The
 [`assert.deepEqual()`][] implementation does not test the
-[`[[Prototype]]`][prototype-spec] of objects, attached symbols, or
-non-enumerable properties — for such checks, consider using
-[`assert.deepStrictEqual()`][] instead. This can lead to some
-potentially surprising results. For example, the following example does not
-throw an `AssertionError` because the properties on the [`RegExp`][] object are
-not enumerable:
+[`[[Prototype]]`][prototype-spec] of objects or enumerable own [`Symbol`][]
+properties — for such checks, consider using [`assert.deepStrictEqual()`][]
+instead. This can lead to some potentially surprising results. For example, the
```

Comment:
Nit: New paragraph starting with `This can lead…`. Maybe also replace `This`. And get rid of `For example` as the next few words are `the following example`. And linkify `AssertionError` maybe? All told, maybe this?:

> `assert.deepEqual()` can have potentially surprising results. The following example does not throw an [`AssertioneError`][] because the properties on the [`RegExp`][] object are not enumerable: